code/controllers/master.dm

```diff
@@ -405,7 +405,7 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 		if (SS_flags & SS_NO_FIRE)
 			subsystemstocheck -= SS
 			continue
-		if (!(SS_flags & SS_TICKER) && (SS_flags & SS_KEEP_TIMING) && SS.last_fire + (SS.wait * 0.75) > world.time)
+		if ((SS_flags & (SS_TICKER|SS_KEEP_TIMING)) == SS_KEEP_TIMING && SS.last_fire + (SS.wait * 0.75) > world.time)
```

Comment:
== SS_KEEP_TIMING ??

Reply:
Yes, cheap and easy way to check for 1 flag and excluding some other flag. if it has both, the result of the `&` will equal SS_TICKER|SS_KEEP_TIMING, if it has none, the result is 0, if it has just one, the result is the flag it has.

Reply:
Its easy to forget that bitwise operators like `&` aren't boolean expressions, they do math on the bits and return the resulting number, in `&`'s case it returns a number with the 1s that exist on both carried over.